src/diffusers/pipelines/deprecated/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -155,6 +154,8 @@ class AltDiffusionPipeline(
             A `CLIPImageProcessor` to extract features from generated images; used as inputs to the `safety_checker`.
     """
 
+    deprecate("AltDiffusionPipeline", "1.0.0", "")
```

Comment:
Actually we won't fully remove the pipeline for now, we'll just stop accepting changes to it. So think we can remove this statement for now